pkg/framework/deployment.go

```diff
@@ -53,7 +53,11 @@ func (c *ContextData) ListPodsForDeployment(deployment *appsv1.Deployment) (*cor
 	return c.Clients.KubeClient.CoreV1().Pods(c.Namespace).List(listOps)
 }
 
-func WaitForStatefulSet(kubeclient kubernetes.Interface, namespace, name string, count int, retryInterval, timeout time.Duration) error {
+func WaitForStatefulSet(kubeclient kubernetes.Interface, namespace, name string, count int, retryInterval, timeout time.Duration) error { // I'd deprecate this method but it might be used in tests etc.
```

Comment:
Do you think it's better idea to mark this function as deprecated (if go can do this) and crate a task fro refactoring?